src/clients/HubPoolClient.ts

```diff
@@ -974,12 +974,7 @@ export class HubPoolClient extends BaseAbstractClient {
       this.proposedRootBundles.push(
         ...events["ProposeRootBundle"]
           .filter((event) => !this.configOverride.ignoredHubProposedBundles.includes(event.blockNumber))
-          .map((event) => {
-            return {
-              ...spreadEventWithBlockNumber(event),
-              transactionHash: event.transactionHash,
```

Comment:
Nice catch - odd that we had this here.

Reply:
I assume this is a holdover from something